python/pyarrow/tests/test_compute.py

```diff
@@ -1794,13 +1794,9 @@ def test_strptime():
 @pytest.mark.skipif(sys.platform == 'win32',
                     reason="Timezone database is not available on Windows yet")
 def test_strftime():
-    from pyarrow.vendored.version import Version
 
     def _fix_timestamp(s):
-        if Version(pd.__version__) < Version("1.0.0"):
-            return s.to_series().replace("NaT", pd.NaT)
-        else:
-            return s
+        return s
```

Comment:
Maybe we ought to remove the calls to this function? I count 7, all in this file.

Reply:
Oh, haven't even realised what I left here 🤦‍♀️  Thanks!